i18n/locales/source/blockly-mooc/applab.json

```diff
@@ -433,6 +433,8 @@
   "stepIn": "Step in",
   "stepOver": "Step over",
   "stepOut": "Step out",
+  "submitYourProject": "Submit your project",
+  "submitYourProjectConfirm": "You cannot edit your project after submitting it, really submit?",
```

Comment:
hm, are these just supposed to be in  apps/i18n/common/en_us.json? @tanyaparker I should revert these, right?